tools/bqtag/bqtag/bqtag.py

```diff
@@ -0,0 +1,800 @@
+            if not self.catalog_project:
+                _, project_id = google.auth.default()
+                self.catalog_project = project_id
+
+        LOGGER.debug("Data Catalog Client Initialised.")
+
+
+    # Create Taxonomy and Tags
+    def create_taxonomy(self, tags: list) -> bool:
+        """
+        Create Taxonomy and Policy Tags
+
+        :params tags: List of Tags to be created. Each tag is
+                      represented as dictionary -
+                      {
+                        "name": "Name of Tag",
+                        "description": "Description of Tag",
+                        "parent_policy_tag": "Parent Policy
+                                              Tag for nested
+                                              policy tags"
+                      }
+
+        :return True if success and False if Failure
+        """
+
+        try:
+            taxonomy = datacatalog_v1.types.Taxonomy()
+            taxonomy.display_name=self.taxonomy
+            created_taxonomy = self.dc.create_taxonomy(
+                                    parent="projects/{p}/locations/{l}".format(
+                                                p=self.catalog_project,
+                                                l=self.location.lower()
+                                            ),
+                                    taxonomy=taxonomy)
+        except (MethodNotImplemented, NotFound) as e:
+            LOGGER.error(
+                            "Could not create Taxonomy. \
+                            API call to Google failed. Error received: %s",
+                            str(e)
+                        )
+            LOGGER.error(traceback.format_exc())
+            return False
+
+        LOGGER.info("Taxonomy Created Successfully.")
+
+        try:
+            for tag in tags:
+                policy_tag = datacatalog_v1.types.PolicyTag()
+                policy_tag.display_name = tag["name"]
+                policy_tag.description = tag.get("description", "")
+                policy_tag.parent_policy_tag = tag.get("parent_policy_tag", "")
+                self.dc.create_policy_tag(parent=created_taxonomy.name,
+                                          policy_tag=policy_tag)
+        except (MethodNotImplemented, NotFound) as e:
+            LOGGER.error(
+                            "Could not create Taxonomy. \
+                            API call to Google failed. Error received: %s",
+                            str(e)
+                        )
+            LOGGER.error(traceback.format_exc())
+            return False
+
+        LOGGER.info("%s Tags Created Successfully.", str(len(tags)))
+
+        return True
+
+
+    # Download policy tags from Data Catalog Taxonomy and save them
+    # in self.policy_tags
+    def fetch_policy_tags(self) -> bool:
+        """
+        Download policy tags from Taxonomy
+        :return True if success and False if Failure
```

Comment:
Missing colon symbol.

Syntax is `:returns:  True if...`

Reply:
Changed to

```
  """
        Download policy tags from Taxonomy
        :return: True if success and False if Failure
        """
        LOGGER.debug("Determining Taxonomy ID using Taxonomy Name provided.")

```